packages/lyrebird/lyrebird.pacscript

```diff
@@ -0,0 +1,36 @@
+name="lyrebird"
+pkgname="lyrebird"
```

Comment:
```suggestion

```
Not needed as there are no extra suffixes like `-git` to this pacscript. And remember to refactor all instances of `pkgname` -> `name`